translations/en.i18n.json

```diff
@@ -1,6 +1,7 @@
 {
   "en": {
     "moderation": {
+      "moderation": "Moderation",
```

Comment:
if this is a new text, please make a ticket regarding it requiring a translation, and link it to Sagi
once translations are ready we will add them in the other i18n files here